javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentTracerProviderConfigurer.java

```diff
@@ -58,4 +65,27 @@ private static boolean loggingExporterIsNotAlreadyConfigured() {
         .getList("otel.traces.exporter", Collections.emptyList())
         .contains("logging");
   }
+
+  private static String getHeliosSamplingRationProperty() {
+    String ratio = System.getenv(String.valueOf(RatioProperty.HS_SAMPLING_RATIO));
+    if (ratio == null) {
+      ratio = System.getProperty(RatioProperty.HS_SAMPLING_RATIO.propertyName());
+    }
+
+    return ratio;
+  }
+
+  private enum RatioProperty {
```

Comment:
Why do we need this enum? Can't we just use final strings?

Reply:
it allows us to use both the name of the enum and it's string property - to allow configure this via runtime config also (not only env var)

Reply:
Not sure I understand why it's better than two final strings